test/classes/delete-free/lifetimes/bad-push-back-borrow.good

```diff
@@ -1,4 +1,5 @@
 bad-push-back-borrow.chpl:20: In function 'addone':
+bad-push-back-borrow.chpl:25: warning: push_back is deprecated - please use List.extend
```

Comment:
These lifetime push-back tests should be updated to test `list`

Reply:
Hmmm, I duplicated this test but using `list` instead, and gave `list.append` the following new signature:

`proc append(pragma "no auto destroy" in: x) lifetime this < x { ... }`

But the test compiles just fine instead of producing an error.

Reply:
OK, then just make the `list` version a `.future` and create a TODO issue to revisit it and assign it to me.